databricks/koalas/tests/test_indexing.py

```diff
@@ -1128,37 +1128,37 @@ def test_index_operator_datetime(self):
 
         # Positional iloc search
         self.assert_eq(kdf[:4], pdf[:4])
-        self.assert_eq(kdf[:3], pdf[:3])
-        self.assert_eq(kdf[3:], pdf[3:])
-        self.assert_eq(kdf[2:], pdf[2:])
-        self.assert_eq(kdf[2:3], pdf[2:3])
-        self.assert_eq(kdf[2:-1], pdf[2:-1])
-        self.assert_eq(kdf[10:3], pdf[10:3])
+        self.assert_eq(kdf[:3], pdf[:3], almost=True)
+        self.assert_eq(kdf[3:], pdf[3:], almost=True)
+        self.assert_eq(kdf[2:], pdf[2:], almost=True)
+        self.assert_eq(kdf[2:3], pdf[2:3], almost=True)
+        self.assert_eq(kdf[2:-1], pdf[2:-1], almost=True)
+        self.assert_eq(kdf[10:3], pdf[10:3], almost=True)
```

Comment:
without `almost=True` here, It will fail in pandas 1.1.1 as below.

```python
Failure
Traceback (most recent call last):
  File "/Users/haejoon.lee/Desktop/git_store/koalas/databricks/koalas/testing/utils.py", line 137, in assertPandasEqual
    pd.util.testing.assert_frame_equal(
  File "/Users/haejoon.lee/opt/miniconda3/envs/koalas-dev-3.8/lib/python3.8/site-packages/pandas/_testing.py", line 1611, in assert_frame_equal
    assert_series_equal(
  File "/Users/haejoon.lee/opt/miniconda3/envs/koalas-dev-3.8/lib/python3.8/site-packages/pandas/_testing.py", line 1327, in assert_series_equal
    assert lidx.freq == ridx.freq, (lidx.freq, ridx.freq)
AssertionError: (None, <Day>)
```